test/fixtures/es-modules/tla/unresolved-withexitcode.mjs

```diff
@@ -1,2 +1,7 @@
+process.on('exit', (exitCode) => {
```

Comment:
Can you put these into separate tests? Otherwise we lose coverage of the cases where there isn't any exit handler installed.

Reply:
Done 🙂